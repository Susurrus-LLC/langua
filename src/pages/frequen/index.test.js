import React from 'react'
import ReactDOM from 'react-dom'
import renderer from 'react-test-renderer'

import { PureFrequen as Frequen } from '.'

describe('Frequen', () => {
  const data = {
    site: {
      siteMetadata: {
        toolInfo: {
          frequen: {
            title: 'Frequen',
            link: 'frequen',
            description: 'Frequen description'
          }
        }
      }
    }
  }

  it('renders without crashing', () => {
    const div = document.createElement('div')
    ReactDOM.render(<Frequen data={data} />, div)
    ReactDOM.unmountComponentAtNode(div)
  })

  it('renders accurately', () => {
    let tree = renderer.create(<Frequen data={data} />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
