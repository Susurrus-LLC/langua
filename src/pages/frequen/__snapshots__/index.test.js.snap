// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Frequen renders accurately 1`] = `
<div
  className="app"
>
  <header
    className="header"
  >
    <div
      className="titleArea"
    >
      <h1
        className="siteTitle"
      >
        <a
          className="titleLink"
          href="/"
        />
      </h1>
      <p
        className="subtitle"
      />
    </div>
    <nav
      className="nav"
    >
      <ul
        className="menu"
      >
        <li
          className="navLink"
        >
          <a
            href="/gen"
          >
            Gen
          </a>
        </li>
        <li
          className="navLink disabled"
        >
          <a
            href="/morph"
          >
            Morph
          </a>
        </li>
        <li
          className="navLink"
        >
          <a
            href="/deriv"
          >
            Deriv
          </a>
        </li>
        <li
          className="navLink"
        >
          <a
            href="/frequen"
          >
            Frequen
          </a>
        </li>
        <li
          className="navLink disabled"
        >
          <a
            href="/tree"
          >
            Tree
          </a>
        </li>
        <li
          className="navLink disabled"
        >
          <a
            href="/lex"
          >
            Lex
          </a>
        </li>
      </ul>
    </nav>
  </header>
  <main
    className="main"
  >
    <a
      className="btn buttonLink"
      href="/frequen/help"
    >
      Help
    </a>
    <h2
      className="toolTitle"
    >
      undefinedFrequen
    </h2>
    <form
      className="form"
      name="frequen-form"
    >
      <div
        className="corpus part"
      >
        <h5
          className="sectionTitle"
        >
          Text Corpus
        </h5>
        <textarea
          aria-label="Text corpus"
          className="corpusInput input"
          id="corpus"
          name="corpus"
          onChange={[Function]}
          value="Nzɔ de’áxa Tsənaxa, ño lo sɛtsa’ɔk, ño lo de’áxa u Kasaga, ño lo mɛya ayru Tol o Əbe kwə, sa nzɔ xa mpaskale: Steyə rabɛ lu ñire lu mvunə poxe o lu pɛlɔ ah lo mɛwɛ axe, maxə ɔra latsi əbɔwə səyo."
        />
      </div>
      <div
        className="consonants part"
      >
        <h5
          className="sectionTitle"
        >
          Consonants
        </h5>
        <input
          aria-label="Consonants"
          className="input"
          id="consonants"
          name="consonants"
          onChange={[Function]}
          type="text"
          value="b/d/f/g/k/kw/l/m/mp/mv/n/nt/nts/nz/ñ/ŋk/ŋkw/p/r/s/t/ts/v/w/x,h/y/’"
        />
      </div>
      <div
        className="vowels part"
      >
        <h5
          className="sectionTitle"
        >
          Vowels
        </h5>
        <input
          aria-label="Vowels"
          className="input"
          id="vowels"
          name="vowels"
          onChange={[Function]}
          type="text"
          value="a,á/ɛ,ɛ́/e,é/ə,ə́/i,í/ɔ,ɔ́/o,ó/u,ú"
        />
      </div>
      <div
        className="control part"
      >
        <div
          className="controlSide controlLeft"
        >
          <div
            className="controlPiece"
          >
            <button
              className="btn neutral"
              onClick={[Function]}
              role="button"
              type="submit"
            >
              Analyze
            </button>
          </div>
          <div
            className="controlPiece"
          >
            <label
              htmlFor="distinguishCase"
            >
              <input
                checked={false}
                id="distinguishCase"
                name="distinguishCase"
                onChange={[Function]}
                type="checkbox"
                value="distinguishCase"
              />
               
              distinguish upper and lower case
            </label>
          </div>
        </div>
        <div
          className="controlSide controlRight"
        >
          <div
            className="controlPiece"
          >
            <button
              className="btn success"
              id="save"
              onClick={[Function]}
              role="button"
              type="button"
            >
              Save
            </button>
          </div>
          <div
            className="controlPiece"
          >
            <button
              className="btn"
              id="open"
              onClick={[Function]}
              role="button"
              type="button"
            >
              Open
            </button>
            <input
              className="hidden"
              id="file"
              name="file"
              onChange={[Function]}
              type="file"
            />
          </div>
        </div>
      </div>
    </form>
    <div
      className="results"
    />
    <div
      className="Toastify"
    />
  </main>
  <footer
    className="footer"
  >
    <p
      className="copyright"
    >
      <a
        href="undefined/blob/master/CHANGELOG.md"
        rel="noopener noreferrer"
        target="_blank"
      >
        Version 
        <i
          className="fas fa-code"
        />
         
      </a>
      . Built by
       
      <a
        rel="noopener noreferrer"
        target="_blank"
      />
      , copyright © 
      2017–2020
       under the
       
      <a
        href="https://choosealicense.com/licenses/agpl-3.0/"
        rel="noopener noreferrer"
        target="_blank"
      >
        AGPL-3.0 
        <i
          className="fas fa-gavel"
        />
         license
      </a>
      .
       
      <a
        rel="noopener noreferrer"
        target="_blank"
      >
        View this project on 
        <i
          className="fab fa-github"
        />
         GitHub.
      </a>
    </p>
  </footer>
</div>
`;
